Libraries/Components/WebView/WebView.android.js

```diff
@@ -58,6 +58,8 @@ class WebView extends React.Component {
     automaticallyAdjustContentInsets: PropTypes.bool,
     contentInset: EdgeInsetsPropType,
     onNavigationStateChange: PropTypes.func,
+    messagingEnabled: PropTypes.bool,
```

Comment:
I have to add this propType here, because it's the only way to say that it's a boolean. Not doing this causes crashes. However, it's not a property for the user. Either we need to hide it in the docs or find another way way to say this is a boolean.